### PR TITLE
modif to docs

### PR DESCRIPTION
**Pull request by jiaminchen-1031**
add missing information for args and deleting useless variables

### CODE REVIEW

File: interpretdl/interpreter/bidirectional_transformer.py [hunk 1]
```diff
@@ -44,9 +43,16 @@ def interpret(self,
...
+                "head" for head-wise, "token" for token-wise. Default: ``head``.
             start_layer (int, optional): Compute the state from the start layer. Default: ``4``.
             steps (int, optional): number of steps in the Riemann approximation of the integral. Default: ``20``.
+            attn_map_name (str, optional): The layer name to obtain the attention weights, head-wise/token-wise.
+                Default: ``^blocks.*.attn.attn_drop$``.
+            attn_v_name (str, optional): The layer name for query, key, value, token-wise.
+                Default: ``blocks.*.attn.qkv``.
+            attn_proj_name (str, optional): The layer name for linear projection, token-wise.
+                Default: ``blocks.*.attn.proj``.
             labels (list or tuple or numpy.ndarray, optional): The target labels to analyze. The number of labels
```
Comment (holyseven) [inline interpretdl/interpreter/bidirectional_transformer.py:56]: -> label

File: interpretdl/interpreter/bidirectional_transformer.py [hunk 2]
```diff
@@ -170,9 +174,18 @@ def interpret(self,
...
             steps (int, optional): number of steps in the Riemann approximation of the integral. Default: ``20``.
+            embedding_name (str, optional): The layer name for embedding, head-wise/token-wise.
+                Default: ``^ernie.embeddings.word_embeddings$``.
+            attn_map_name (str, optional): The layer name to obtain the attention weights, head-wise/token-wise.
+                Default: ``^ernie.encoder.layers.*.self_attn.attn_drop$``.
+            attn_v_name (str, optional): The layer name for value projection, token-wise.
+                Default: ``^ernie.encoder.layers.*.self_attn.v_proj$``.
+            attn_proj_name (str, optional): The layer name for linear projection, token-wise.
+                Default: ``ernie.encoder.layers.*.self_attn.out_proj$``.
             labels (list or tuple or numpy.ndarray, optional): The target labels to analyze. The number of labels
```
Comment (holyseven) [inline interpretdl/interpreter/bidirectional_transformer.py:189]: -> label

File: interpretdl/interpreter/bidirectional_transformer.py (hunk 1 above)
Comment (holyseven) [reply interpretdl/interpreter/bidirectional_transformer.py:56]: generic_attention.py 也有需要修改的地方

Comment (jiaminchen-1031) [reply interpretdl/interpreter/bidirectional_transformer.py:56]: ok

**Pull request merged**